Game/Source/GDKTestGyms/Public/UserExperienceComponent.h

```diff
@@ -0,0 +1,59 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "Components/ActorComponent.h"
+#include "UserExperienceComponent.generated.h"
+
+DECLARE_LOG_CATEGORY_EXTERN(LogUserExperienceComponent, Log, All);
+
+struct ObservedUpdate
+{
+	float Value{ 0.0f };
+	float TimeSinceChange{ 0.0f };
+	TArray<float> TrackedChanges;
+};
+
+UCLASS( ClassGroup=(Custom), meta=(BlueprintSpawnableComponent) )
+class GDKTESTGYMS_API UUserExperienceComponent : public UActorComponent
```

Comment:
Can we get a comment about what this component is, and how it's supposed to work?